src/connected/connectionsetup.ts

```diff
@@ -173,7 +174,8 @@ function renderConnectionSetupPanel(context: vscode.ExtensionContext, webview: v
         </div>
         ${renderOrganizationKeyField(initialState)}
         <vscode-text-field id="connectionId" type="text" placeholder="My ${serverProductName} Connection" size="40"
-          title="Optionally, please give this connection a memorable name" value="${initialConnectionId}"
+          title="Optionally, please give this connection a memorable name. If no name is provided, Sonar will autmatically generate one." 
```

Comment:
small typo -should be `automatically`. But TBH I'm not sure if this word is needed? Maybe simply `SonarLint will generate one` is OK.